scripts/dockerfiles/monitrc.erb

```diff
@@ -14,3 +14,6 @@ include /etc/monit/monitrc.d/rsyslog
 
 include /var/vcap/monit/*.monitrc
 include /var/vcap/monit/job/*.monitrc
+
+# Standard pseudo-job. Triggered to mark the role as ready.
+check file ready path /tmp/ready
```

Comment:
Ok. The key point for me is __we install__. And like are using `/var/vcap/monit` already.
Then I see no problem to use that location also.
If there is a conflict in the future it will be there anyway.